configuration/restdocs.md

```diff
@@ -7,14 +7,29 @@ title: openHAB REST API
 
 # openHAB REST API
 
-The REST API of openHAB serves different purposes.
-It can be used to integrate openHAB with other systems as it allows read access to items and item states as well as status updates or the sending of commands for items.
-Furthermore, it gives access to the sitemaps, so that it is the interface to be used by remote user interfaces (e.g. fat clients or javascript based web clients).
-The REST API also supports server-push, so you can subscribe yourself on change notification for certain resources.
+The REST API gives the ability to access a wide range of information associated with a running openHAB installation, and the ability to invoke changes to the openHAB installation. 
```

Comment:
"access a wide range of information associated with a running openHAB installation"

this seems very generic. How about a simple first sentence like (not polished):

> The openHAB [REST API](https://en.wikipedia.org/wiki/REST_API) offers access to most aspects of the openHAB system. Data related to, e.g., Items, Things and Bindings can be accessed and actions can be performed via a simple http based interaction method.

Reply:
Or, how about this...
The [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) API provides both the ability to access a wide range of information associated with a running openHAB installation, and the ability to invoke changes to the openHAB installation.
This summarizes both the ability to access information and set information.  It also provides a link to a wikipedia article on REST, allowing us to cut out the definition of REST further down in the article.

Reply:
sounds good, done; adding the link to wikipedia....but am I remembering correctly that links to sources outside the OH universe are discouraged? At least, I thought reading it...which is why I added some description in the first place. Linking it is more elegant though.....